exist-core/src/main/java/org/exist/util/SaxonConfigurationHolder.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * eXist-db Open Source Native XML Database
+ * Copyright (C) 2001 The eXist-db Authors
+ *
+ * info@exist-db.org
+ * http://www.exist-db.org
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.exist.util;
+
+import com.evolvedbinary.j8fu.lazy.AtomicLazyVal;
+import net.sf.saxon.lib.Feature;
+import net.sf.saxon.s9api.Processor;
+import net.sf.saxon.trans.XPathException;
+import org.exist.storage.BrokerPool;
+import org.jline.utils.Log;
+
+import javax.xml.transform.stream.StreamSource;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public final class SaxonConfigurationHolder {
+
+  public final static String CONFIGURATION_ELEMENT_NAME = "saxon";
```

Comment:
Generic name of static export should be avoided.
see also comment from @adamretter in #4838

Reply:
I've prefixed all the statics with SAXON_